National Parks/National Parks/Views/SigninView.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  SigninView.swift
+//  National Parks
+//
+//  Created by Michael Koohang on 1/21/20.
+//  Copyright © 2020 Will Said. All rights reserved.
+//
+
+import SwiftUI
+
+
+struct SigninView: View {
```

Comment:
Why are you creating a new SigninView? What new functionalities does it add? It seems to be duplicating the code from the SignupView

Reply:
I created a new view because I thought the sign in and sign up pages should be two distinct pages, rather than one class that is either a sign in or a sign up page. It was easier for me to manipulate them separately and add functionality to buttons that were distinct to each page. I could go back and try to keep it in one class, but the short term solution seemed faster. There is no new functionality, just separated.